v2/internal/testcommon/vcr/redact.go

```diff
@@ -147,7 +155,7 @@ func hideKubeConfigs(s string) string {
 var keyMatcher = regexp.MustCompile("(?:[A-Za-z0-9+/]{4}){10,}(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)")
 
 func hideKeys(s string) string {
-	return keyMatcher.ReplaceAllLiteralString(s, "{KEY}")
+	return keyMatcher.ReplaceAllLiteralString(s, "{REDACTED}")
```

Comment:
Ditto here - we used `KEY` for a reason, so we know what kind of value was removed.